src/messages.js

```diff
@@ -22,4 +22,7 @@ export default {
 
   'vocab.citation.local.name': 'Citations',
   'vocab.citation.collectionName.name': 'Citations',
+
+  'vocab.concept.associated.name': 'Subjects',
+  'vocab.concept.collectionName.name': 'Subjects',
 };
```

Comment:
I think you mean vocab.concept.associated.collectionName. Please make the name "Subject". As a general rule, name should be singular, collectionName plural. I didn't catch this with Citation, but same thing there.